include/onnxruntime/core/providers/openvino/openvino_provider_factory.h

```diff
@@ -5,13 +5,23 @@
 
 #ifdef __cplusplus
 extern "C" {
+#else
+#include <stdbool.h>
 #endif
 
 /**
- * \param device_id openvino device id, starts from zero.
+ * \param device_type openvino device type and precision. Could be any of
+ * CPU_FP32, GPU_FP32, GPU_FP16, MYRIAD_FP16, VAD-M_FP16 or VAD-F_FP32.
  */
 ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_OpenVINO,
-    _In_ OrtSessionOptions* options, const char* device_id);
+    _In_ OrtSessionOptions* options, _In_ const char* device_type);
+
+/**
+ * \param settings_str string of Key-Value pairs with '\n' used to delimit
+ * pairs and '|' used to delimit key and value within a pair.
+ */
+ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_OpenVINOEP,
```

Comment:
not particularly fond of adding EP to make the new name.
but I understand we're kind of limited here. maybe it's better to add a _2 to it as sort of a version marker. 
OrtSessionOptionsAppendExecutionProvider_OpenVINO_2 
@pranavsharma , any guidance here?

Reply:
For DML, we added Ex to the API name to indicate 'extended' API. https://github.com/microsoft/onnxruntime/blob/master/include/onnxruntime/core/providers/dml/dml_provider_factory.h#L42